### PR TITLE
#DVFD9-46 and #DVFD9-42 Fix Remove Show table button and Duplicate Data Shown on the Bar Chart

### PR DESCRIPTION
**Pull request by WilsonWu2100**
Remove Show Table button, since it's not working due to the missing header field for the table when showing as a bar chart

### CODE REVIEW

File: js/jquery.dvfTables.js [hunk 1]
```diff
@@ -111,14 +111,6 @@
         return this;
       }
 
-      $('<button/>')
```
Comment (thery-dh) [inline js/jquery.dvfTables.js]: @WilsonWu2100 we dot simply want to remove the button but instead remove it when the required configuration is missing
cc @jez500

**Review by thery-dh — CHANGES_REQUESTED** [review]

**Comment by jez500** [comment]
@thery-dh can this be closed, looks like its been merged elsewhere?

**Comment by WilsonWu2100** [comment]
Sorry @jez500, please do not close this branch, I just reverted my commit and might push again

File: js/jquery.dvfTables.js (hunk 1 above)
Comment (WilsonWu2100) [reply js/jquery.dvfTables.js]: This issue was fixed now

**Comment by WilsonWu2100** [comment]
This fix can resolve two issues, one is Show Button not working, the other is Duplicate data shown on Bar Chart

File: src/Plugin/Visualisation/Style/VisualisationStyleBase.php [hunk 2]
```diff
@@ -365,7 +365,7 @@ public function getSourceRecords() {
    *   The fields.
    */
   protected function fields() {
-    return array_filter($this->config('data', 'fields'));
+    return array_unique(array_filter($this->config('data','fields')));
```
Comment (jez500) [inline src/Plugin/Visualisation/Style/VisualisationStyleBase.php]: please add the space back between 'data', and 'fields'

**Review by jez500 — APPROVED** [review]
Just a minor CS fix, other than that good to go

File: src/Plugin/Visualisation/Style/VisualisationStyleBase.php (hunk 2 above)
Comment (WilsonWu2100) [reply src/Plugin/Visualisation/Style/VisualisationStyleBase.php]: Done

**Pull request merged**